pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env nextflow
+/** See the NOTICE file distributed with this work for additional information
+* regarding copyright ownership.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+nextflow.enable.dsl=2
+
+include { listSubDirs; ensemblLogo } from './../utilities.nf'
+
+def helpMessage() {
+    log.info ensemblLogo()
+    log.info """
+    Usage:
+
+    """.stripIndent()
+}
+
+params.help = ''
+if (params.help) {
+    helpMessage()
+    exit 0
+}
+
+if (!params.dir) {
+    helpMessage()
+    exit 0
+}
+
+process prepareBusco {
+    input:
+        path protfa
+    output:
+        path "longest_busco_proteins.fas"
+    script:
+    """
+    ${params.scriptsDir}/filter_for_longest_busco.py -i $protfa -o longest_busco_proteins.fas
```

Comment:
Ideally, to follow a sort of "compara convention" this script should be stored in the nextflow.config under something like:
```
longest_busco_filter_exe = "$ENSEMBL_ROOT_DIR/ensembl-compara/src/python/scripts/filter_for_longest_busco.py"
```
under `params {`
And then 
```suggestion
    ${params.longest_busco_filter_exe} -i $protfa -o longest_busco_proteins.fas
```